conf/cassandra.yaml

```diff
@@ -2113,3 +2110,16 @@ accord_transactions_enabled: false
 #   a stable cluster. If a node from a previous version was started by accident we won't any longer toggle behaviors as when UPGRADING.
 #
 storage_compatibility_mode: NONE
+
+# accord:
+# Enables the execution of Accord (multi-key) transactions on this node.
```

Comment:
spacing is off, and should we add a `#` in front so users can block uncomment and have this still work?  We don't really have many examples of nested properties with comments inside the nesting... so this could be the first time this could cause confusion I think.

Reply:
there is one: `repair` section but it is broken as well. 
will fix the accord section